tests/testthat/test-prepare_pctr_product.R

```diff
@@ -98,3 +98,30 @@ test_that("handles tiltIndicator output", {
     )
   )
 })
+
+test_that("*matching_certainty_company_average column yield only 1 unique value", {
+  product_empty <- tibble(
+    companies_id = "a",
+    grouped_by = NA_character_,
+    risk_category = NA_character_,
+    activity_uuid_product_uuid = NA_character_,
+    co2_footprint = NA_character_,
+    tilt_sector = NA_character_,
+    tilt_subsector = NA_character_,
+    isic_4digit = NA_character_
+  )
```

Comment:
So this test is only for the case when the output of tiltIndicator at product level is empty? 

If so we need to reflect that in the title of the test. For example:

"with empty tiltIndicator output at product level yields, `matching_certainty_company_average` yields a single unique value per `company_id`"